From 42593cded8643f4bd365a51c7d83fec601e49167 Mon Sep 17 00:00:00 2001
Message-ID: <42593cded8643f4bd365a51c7d83fec601e49167.1747662482.git.michal.simek@amd.com>
From: Michal Simek <michal.simek@amd.com>
Date: Fri, 25 Apr 2025 09:24:48 +0200
Subject: [PATCH] riscvXX-softmmu: Enable emaclite driver for MB-V

Emaclite can be also wired with MB-V that's why enable it by default.

Signed-off-by: Michal Simek <michal.simek@amd.com>
Signed-off-by: Francisco Iglesias <francisco.iglesias@amd.com>
---
 configs/devices/riscv32-softmmu/default.mak | 1 +
 configs/devices/riscv64-softmmu/default.mak | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/devices/riscv32-softmmu/default.mak b/configs/devices/riscv32-softmmu/default.mak
index c6c3dcd4431a..7845858ae87c 100644
--- a/configs/devices/riscv32-softmmu/default.mak
+++ b/configs/devices/riscv32-softmmu/default.mak
@@ -33,6 +33,7 @@ CONFIG_XILINX_IOMODULE_PIT=y
 CONFIG_XILINX_IOMODULE_GPI=y
 CONFIG_XILINX_IOMODULE_GPO=y
 CONFIG_XILINX_IOMODULE_INTC=y
+CONFIG_XILINX_ETHLITE=y
 
 CONFIG_XLNX_ZYNQMP=y
 CONFIG_XLNX_VERSAL=y
diff --git a/configs/devices/riscv64-softmmu/default.mak b/configs/devices/riscv64-softmmu/default.mak
index 4886ca35dcb7..4f5528eeda9a 100644
--- a/configs/devices/riscv64-softmmu/default.mak
+++ b/configs/devices/riscv64-softmmu/default.mak
@@ -20,6 +20,7 @@ CONFIG_XILINX=y
 CONFIG_SSI=y
 CONFIG_I2C=y
 CONFIG_XILINX_AXI=y
+CONFIG_XILINX_ETHLITE=y
 CONFIG_XILINX_SPI=y
 CONFIG_XILINX_SPIPS=y
 CONFIG_PTIMER=y
-- 
2.43.0

